elastiknn-testing/src/test/scala/com/klibisz/elastiknn/mapper/VectorMapperSuite.scala

```diff
@@ -1,28 +1,26 @@
 package com.klibisz.elastiknn.mapper
 
-import java.util.UUID
-
-import com.klibisz.elastiknn.api.{ElasticsearchCodec, Mapping, NearestNeighborsQuery, Vec}
+import com.klibisz.elastiknn.api.{Mapping, NearestNeighborsQuery, Vec, XContentCodec}
 import com.klibisz.elastiknn.testing.{Elastic4sMatchers, ElasticAsyncClient}
 import com.sksamuel.elastic4s.ElasticDsl._
 import com.sksamuel.elastic4s.requests.get.GetResponse
 import com.sksamuel.elastic4s.requests.indexes.IndexRequest
 import com.sksamuel.elastic4s.{Indexes, Response}
+import io.circe.JsonObject
 import io.circe.parser.parse
-import io.circe.{Json, JsonObject}
 import io.circe.syntax._
-import org.scalatest._
-import org.scalatest.funsuite.AsyncFunSuite
+import org.scalatest.freespec.AsyncFreeSpec
 import org.scalatest.matchers.should.Matchers
 
+import java.util.UUID
 import scala.concurrent.Future
 import scala.util.Random
 
-class VectorMapperSuite extends AsyncFunSuite with Matchers with Inspectors with Elastic4sMatchers with ElasticAsyncClient {
+class VectorMapperSuite extends AsyncFreeSpec with Matchers with Elastic4sMatchers with ElasticAsyncClient {
```

Comment:
Main change here is no longer using `Inspectors` and `forAll` because it was making test failures impossible to read.